sql/xsql-shell/src/main/scala/org/apache/spark/sql/xsql/shell/SparkXSQLShell.scala

```diff
@@ -227,11 +227,10 @@ object SparkXSQLShell extends Logging {
           }
           if (!silent && !sql.toLowerCase.startsWith("add") && !sql.toLowerCase.startsWith("set")) {
             //modify for spark2.3 , to first start local , then restart scheduler
-            if (!restarted && sql.toLowerCase.contains("select ") &&
+            if (!restarted && (sql.toLowerCase.contains("select ") ||
+                sql.toLowerCase.contains("select\\n")) &&
```

Comment:
Why we not write like this:
`if (!restarted && sql.toLowerCase.contains("select")`